### PR TITLE
02 css ecommerce products

### PR DESCRIPTION
**Pull request by mmusk**
feat: added Products page to this website

### CODE REVIEW

File: tasks-by-users/mmusk/02-second-task-css/style.css [hunk 1]
```diff
@@ -99,114 +98,296 @@ p {
...
+.footer h3 {
+  color: #fff;
+
+  margin-bottom: 20px;
+}
+
+.footer-col-1,
+.footer-col-2,
+.footer-col-3,
+.footer-col-4 {
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/02-second-task-css/style.css:254]: can you use attribute selector here @mmusk

File: tasks-by-users/mmusk/02-second-task-css/style.css [hunk 2]
```diff
@@ -99,114 +98,296 @@ p {
...
-    box-shadow: 0 0 20px 0px rgba(0, 0, 0, 0.1);
     
-    transition: transform 0.5s; 
+  }
+  nav ul li {
+    display: block;
+
+    margin-right: 50px;
+    margin-top: 10px;
+    margin-bottom: 10px;
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/02-second-task-css/style.css:331]:
```suggestion
margin: 10px 50px 10px 0;
```

File: tasks-by-users/mmusk/02-second-task-css/style.css [hunk 3]
```diff
@@ -99,114 +98,296 @@ p {
...
+  }
+}
+
+@media only screen and (max-width: 00px) {
+  .row {
+    text-align: center;
+  }
+  .col-2,
+  .col-3,
+  .col-4 {
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/02-second-task-css/style.css:349]: attribute selector here

File: tasks-by-users/mmusk/02-second-task-css/style.css [hunk 4]
```diff
@@ -99,114 +98,296 @@ p {
...
-.small-container{
-    max-width: 1080px;
-    margin: auto;
-    padding-left: 25px;
-    padding-right: 25px;
+.small-container {
+  max-width: 1080px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
```
Comment (coder-mezab) [inline tasks-by-users/mmusk/02-second-task-css/style.css:120]:
```suggestion
  padding: 0 25px;
```

**Comment by coder-mezab** [comment]
requested some changes @mmusk once done mention me here i will approve

File: tasks-by-users/mmusk/02-second-task-css/style.css (hunk 1 above)
Comment (mmusk) [reply tasks-by-users/mmusk/02-second-task-css/style.css]: fixed @coder-mezab

File: tasks-by-users/mmusk/02-second-task-css/style.css (hunk 3 above)
Comment (mmusk) [reply tasks-by-users/mmusk/02-second-task-css/style.css]: fixed @coder-mezab

**Review by coder-mezab — APPROVED** [review]
lgtm Approving

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Review by shaikh-7abish — APPROVED** [review]

**Pull request merged**